modules/core-services/services/security/process.go

```diff
@@ -267,9 +267,10 @@ func (a AppPermissionProcess) GetAllRoles(ctx context.Context) ([]string, error)
 	}
 	// get all project-level application
 	_, data, err := a.Adaptor.Db.GetApplicationsByIDs(&dto.OrgID, &dto.ProjectID, nil, &apistructs.ApplicationListRequest{
```

Comment:
already pass &dto.ProjectID, is it nessasery to construct ApplicationListRequest with ProjectID?

Reply:
we are not using the second params (&dto.ProjectID) , will remove it in future.

Reply:
okay